thirdeye/thirdeye-frontend/app/pods/components/self-serve-graph/template.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="col-xs-12 te-graph-container">
+  <div class="te-graph-alert {{if (or isMetricDataLoading isMetricDataInvalid) 'te-graph-alert--pending'}}">
+    {{#if isDimensionFetchDone}}
+      <div class="te-form__super-label">...Displaying top 5 contributing subDimensions for <strong>{{alertDimension}}</strong></div>
+    {{/if}}
+    {{#if isMetricDataLoading}}
+        <div class="spinner-wrapper--self-serve">{{ember-spinner}}</div>
```

Comment:
fix: indentation + BEM. 
modifier classes in BEM are `augmenting` a base class.
 based on what you have the correct way would be `class="""spinner-wrapper spinner-wrapper--self-serve"`

Reply:
sure thing. thanks.